projects/angular/components/ui-grid/src/components/ui-grid-toggle-columns/ui-grid-toggle-columns.component.scss

```
@@ -1,33 +1,54 @@
 $toggle-col-width: 10em;
 
 .ui-grid-toggle-columns {
-    margin-right: 6px;
-    padding-left: 0;
-
-    mat-select {
-        overflow: hidden;
-        width: 0;
-        // adjustments in order to match trigger icon with optgroup-label icon
-        position: relative;
-        top: -10px;
-        left: -4px;
-    }
+  margin-right: 6px;
+  padding-left: 0;
+
+  mat-select {
+    overflow: hidden;
+    width: 0;
+    // adjustments in order to match trigger icon with optgroup-label icon
+    position: relative;
+    top: -10px;
+    left: -4px;
+  }
+}
+
+.alternatie-mat-select {
+  top: -8px !important;
+  left: -82px !important;
 }
 
 // overlay panel
 .ui-grid-toggle-panel {
-    // IE fix - min-width with calc does not render properly so we add a fallback in case min-width is evaluated even lower
-    width: 210px !important;
-
-    .mat-optgroup-label {
-        padding-left: 12px;
-        display: flex;
-        padding-right: 6px;
-        justify-content: space-between;
-        align-items: center;
-
-        .mat-icon {
-            margin-right: 0;
-        }
+  // IE fix - min-width with calc does not render properly so we add a fallback in case min-width is evaluated even lower
+  width: 210px !important;
+
+  .mat-optgroup-label {
+    padding-left: 12px;
+    display: flex;
+    padding-right: 6px;
+    justify-content: space-between;
+    align-items: center;
+
+    .mat-icon {
+      margin-right: 0;
+    }
+  }
+}
+
+.ui-grid-alternate-toggle-panel {
```

let's compose this in `.ui-grid-toggle-panel` and override only what's necessary.
```suggestion
&-alternate {
```